R/sub.r

```diff
@@ -15,6 +15,8 @@
 #'   matrix to `start`.
 #'
 #'   Negative values count backwards from the last character.
+#' @param omit_na single logical value; if \code{TRUE}, missing values in any of the arguments
+#' provided will result in an unchanged input; replacement function only
```

Comment:
Missing indent here

Reply:
I think you can remove the bit about replacement function only. It would be useful to have a bit more motivation about when this is useful